app/assets/javascripts/stores/questionnaire_store.cjsx

```diff
@@ -195,11 +195,29 @@ class QuestionnaireStore extends EventEmitter
   setPath: (path) ->
     window.location = path
 
-  setNotification: (msg) ->
+  setNotification: (key, msg) ->
+    title_icon = switch key
+                    when "success" then "check"
+                    when "error" then "times-circle"
+                    else "info-circle"
+    title = key.charAt(0).toUpperCase() + key.slice(1)
+
     $notificationEl = $('.js-questionnaire-notification')
-    $notificationEl.html(msg).fadeIn()
+    $notificationEl.append(
+      $('<div class="alert alert-' + key + '"></div>').append(
+        $('<p class="alert__close"><i class="fa fa-close"></i> Close</p>')
+      ).append(
+        $('<h4 class="alert__title"><i class="fa fa-' + title_icon + '"></i> ' + title + '</h4>')
+      ).append(
+        $('<p class="alert__body">' + msg + '</p>')
+        )
+      )
 
-    setTimeout((-> $notificationEl.fadeOut()), 3000)
+    $notificationEl.fadeIn()
+    setTimeout((-> $notificationEl.fadeOut()), 10000)
```

Comment:
@lucidstack We couldn't decide, 1s didn't seem long enough for longer messages. But 10s is probably too long.

Maybe we could change to something like 3s and then revisit later?